light-zk.js/src/account.ts

```diff
@@ -85,6 +87,7 @@ export class Account {
         secretKey: new Uint8Array(32),
       };
```

Comment:
we should add the hashingSecret here as well, I think this is also the only place where the passed in aes secret is used.
Correct me if I am wrong :)

Reply:
I'll ruminate on it but I the gut feeing is yes, it makes sense to add it

Reply:
agreed it's shouldn't be part of the encryption keypair, I just commented at that line because github doesn't allow me to comment above, it's too far from changes

Reply:
cool, makes sense!